Examples/Io/Root/include/ActsExamples/Io/Root/RootTrackParameterWriter.hpp

```diff
@@ -52,19 +73,33 @@ class RootTrackParameterWriter final : public TrackParameterWriter {
   /// @param [in] trackParams are parameters to write
   ProcessCode writeT(
       const AlgorithmContext& ctx,
-      const std::vector<BoundTrackParameters>& trackParams) final override;
+      const TrackParametersContainer& trackParams) final override;
 
  private:
   Config m_cfg;             ///< The config class
   std::mutex m_writeMutex;  ///< Mutex used to protect multi-threaded writes
   TFile* m_outputFile{nullptr};  ///< The output file
   TTree* m_outputTree{nullptr};  ///< The output tree
   int m_eventNr{0};              ///< the event number of
-  float m_d0{0.};                ///< transversal IP d0
-  float m_z0{0.};                ///< longitudinal IP z0
-  float m_phi{0.};               ///< phi
-  float m_theta{0.};             ///< theta
-  float m_qp{0.};                ///< q/p
+
+  float m_loc0{0.};   ///< loc0
+  float m_loc1{0.};   ///< loc1
+  float m_phi{0.};    ///< phi
+  float m_theta{0.};  ///< theta
+  float m_qop{0.};    ///< q/p
+  float m_time{0.};   ///< time
+  float m_p{0.};      ///< p
+  float m_pt{0.};     ///< pt
+  float m_eta{0.};    ///< eta
+
+  int m_t_charge{0};            ///< Truth particle charge
+  float m_t_loc0{-99.};         ///< Truth parameter loc0
+  float m_t_loc1{-99.};         ///< Truth parameter loc1
+  float m_t_phi{-99.};          ///< Truth parameter phi
+  float m_t_theta{-99.};        ///< Truth parameter theta
+  float m_t_qop{-99.};          ///< Truth parameter qop
+  float m_t_time{-99.};         ///< Truth parameter time
```

Comment:
I'm a fan of using NaN to indicate *empty* values, but that's just my preference.

Reply:
That`s a good idea. I used the NaN for them now.